src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -156,6 +98,8 @@ TER Transactor::payFee ()
     mSourceBalance -= saPaid;
     mTxnAccount->setFieldAmount (sfBalance, mSourceBalance);
 
+    // VFALCO Should we call view().destroyCoins() here as well?
```

Comment:
Why not move the call to `destroyCoins` to this function, since it pays the fee?